### DISCUSSION TITLE
关于“鼠标悬停在译文上时显示原文”有个小延时

### DISCUSSION THREAD

**Discussion by 6Tom** [chunibyocola/sc-translator-crx#77] [General]
首先感谢大佬的辛勤工作和无私分享，扩展很好用，可自定义的条目非常丰富

我有个小问题，就是“鼠标悬停在译文上时显示原文”，会有个小延时，要过上那么一下，才出原文弹窗

所以我是想问这个是特意加的吗，如果是的话可否取消，或者开放一个延时的时长设置？

如果是技术层面的啥问题，就不必折腾了，祝大佬生活愉快

**Comment by chunibyocola** [comment]
> 我有个小问题，就是“鼠标悬停在译文上时显示原文”，会有个小延时，要过上那么一下，才出原文弹窗
>
> 所以我是想问这个是特意加的吗

是的，这个延时是特意加的。具体表现为鼠标悬停在同一段译文上 1000 毫秒（1 秒）后展示原文框，鼠标离开原文框以及所有译文 500 毫秒后隐藏原文框。这个延时的存在是必要的，原文框需要足够大才同时具有美观以及展示大量文本的特性，如果没有这个延时，用户在使用网页的过程中会非常烦恼。

> 如果是的话可否取消，或者开放一个延时的时长设置？

取消的话是不应该的，理由我在上方解析过。对于延时的时长设置我觉得也没有必要， 1 秒整已经是刚刚合适，不太会影响用户操作，也不会让用户等太久。

感谢你的支持

**Reply by 6Tom** [reply]
虽然我还是嫌那1秒比较煎熬，但是真的非常感谢大佬抽空回复，以及耐心的解释
